addons/l10n_latam_check/views/account_payment_view.xml

```diff
@@ -40,7 +40,8 @@
                 <separator/>
                 <filter string="Checks on hand" name="checks_on_hand"
                     domain="[('state', '=', 'posted'),
-                                ('l10n_latam_check_current_journal_id.inbound_payment_method_line_ids.payment_method_id.code', '=', 'in_third_party_checks')]"/>
+                                ('l10n_latam_check_current_journal_id.inbound_payment_method_line_ids.payment_method_id.code', 'in', ['new_third_party_checks', 'in_third_party_checks']),
+                                ('l10n_latam_check_current_journal_id.type', '=', 'cash')]"/>
```

Comment:
@pablohmontenegro is type cash just for safety or why do we add that condition?

Reply:
@pablohmontenegro I agree with jos. Shouldn't be needed the type = cash

Reply:
Thanks @jco-odoo . Can you review this pr again?